admin/load.php

```diff
@@ -156,17 +156,38 @@ function perflab_render_modules_page_field( $module_slug, $module_data, $module_
 				<input type="checkbox" id="<?php echo esc_attr( "{$base_id}_enabled" ); ?>" aria-describedby="<?php echo esc_attr( "{$base_id}_description" ); ?>" disabled>
 				<input type="hidden" name="<?php echo esc_attr( "{$base_name}[enabled]" ); ?>" value="<?php echo $enabled ? '1' : '0'; ?>">
 				<?php
+				if ( 'database/sqlite' === $module_slug && ! wp_is_writable( WP_CONTENT_DIR ) ) {
+					printf(
+						/* translators: %s: WP_CONTENT_DIR */
+						__( 'The SQLite module cannot be activated because the %s directory is not writable.', 'performance-lab' ),
+						WP_CONTENT_DIR
+					);
+				} elseif ( 'database/sqlite' === $module_slug && ( ! extension_loaded( 'sqlite3' ) || ! class_exists( 'SQLite3' ) ) ) {
+					_e( 'The SQLite module cannot be activated because the SQLite extension is not loaded.', 'performance-lab' );
+				} else {
 					printf(
 						/* translators: %s: module name */
 						__( '%s is already part of your WordPress version and therefore cannot be loaded as part of the plugin.', 'performance-lab' ),
 						esc_html( $module_data['name'] )
 					);
+				}
 				?>
 			<?php } ?>
 		</label>
 		<p id="<?php echo esc_attr( "{$base_id}_description" ); ?>" class="description">
 			<?php echo esc_html( $module_data['description'] ); ?>
 		</p>
+		<?php if ( $enabled && ! empty( $module_data['enabled-notice'] ) ) : ?>
+			<?php // Don't use the WP notice classes here, as that makes them move to the top of the page. ?>
+			<p style="background:#fff;border:1px solid #c3c4c7;border-left-width: 4px;border-left-color:#dba617;box-shadow: 0 1px 1px rgba(0, 0, 0, 0.04);padding:1em;max-width:50em;">
+				<?php echo esc_html( $module_data['enabled-notice'] ); ?>
+			</p>
+		<?php elseif ( ! $enabled && perflab_can_load_module( $module_slug ) && ! empty( $module_data['disabled-notice'] ) ) : ?>
+			<?php // Don't use the WP notice classes here, as that makes them move to the top of the page. ?>
+			<p style="background:#fff;border:1px solid #c3c4c7;border-left-width: 4px;border-left-color:#dba617;box-shadow: 0 1px 1px rgba(0, 0, 0, 0.04);padding:1em;max-width:50em;">
+				<?php echo esc_html( $module_data['disabled-notice'] ); ?>
+			</p>
+		<?php endif; ?>
```

Comment:
See my other comments, I would suggest we hardcode the messages here for only the `database/sqlite` module.

Reply:
Done